packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -2137,10 +2137,10 @@ module.exports = function (config, DB) {
           })
           .then((session) => {
             // Returns verified session
-            assert.equal(!! session.mustVerify, false, 'mustVerify is false')
-            assert.equal(session.tokenVerificationId, null, 'tokenVerificationId is not set')
-            assert.equal(session.tokenVerificationCodeHash, null, 'tokenVerificationCodeHash is not set')
-            assert.equal(session.tokenVerificationCodeExpiresAt, null, 'tokenVerificationCodeExpiresAt is not set')
+            assert.isFalse(!! session.mustVerify)
+            assert.isNull(session.tokenVerificationId)
```

Comment:
It's actually `null` rather than undefined?

Reply:
Yep, that's why the tests pass. 😄